docs/kh2/file/type/mset.md

```diff
@@ -0,0 +1,81 @@
+# [Kingdom Hearts II](../../index.md) - MSET (Moveset)
+
+The MSET files contain animations and the effects related to them (I-frames, hitboxes, effects...)
+It is a [BAR file](bar.md) that contains a list of "slots". Each slot may or may not have a "motion".
+The various actions in the game point to a specific motion by a slot position in this list, thus many of the motions are dummies, since the characters only use some of them.
+Eg: For Sora, slot 0 is idle, slot 4 is walking and slot 604 is the basic attack.
+
+Each motion is a BAR file with 2 entries, the model animation and the effects.
+
+## Animation structure
+
+???
```

Comment:
I have found existing `.anb` documentation by luck. It says about solely Animation structure so it'll be able to link it.

```
## Animation structure

See [ANB](../anb/anb.md) documentation.
```

Reply:
Awesome! I didn't see that! I've added the link as suggested. Thanks!